src/features/user/RegisterTrees/RegisterTrees/SingleContribution.tsx

```diff
@@ -7,12 +7,23 @@ import UploadImages from './UploadImages';
 import { useTranslation } from 'next-i18next';
 import formatDate from '../../../../utils/countryCurrency/getFormattedDate';
 import { Button } from '@mui/material';
+import { Image } from '@planet-sdk/common';
 
-interface Props {
-  token: any;
-  contributionGUID: any;
-  contribution: any;
-  slug: any;
+export interface ContributionProperties {
+  contributionImages: Image[];
+  id: string;
+  plantDate: string;
+  plantProject: string | null;
+  treeClassification: string | null;
+  treeCount: number;
+  treeScientificName: string | null;
+  treeSpecies: string;
+}
```

Comment:
In

```<StaticMap geoJson={contribution.geometry} />```

I see the message: `Property 'geometry' does not exist on type 'ContributionProperties'.`. Does the `ContributionProperties` type need to be adjusted?